### PR TITLE
Ensure the dtype for the ids

### PR DESCRIPTION
**Pull request by tomdele**
It is quite unsafe to the use uint64 for the ids due to the numpy issue:
https://github.com/numpy/numpy/issues/15084

Where :
numpy.uint64 + int --> float64
numpy.uint64 += int --> float64

So it fixes the ids dtype to int64 until the issue in numpy is fixed.

- add a IDS_DTYPE to contain the ids dtype
- add a ensure_ids function
- add ensure_ids when ids are returned

### CODE REVIEW

**Comment by tomdele** [comment]
(ping @eleftherioszisis for archngv)

**Comment by asanin-epfl** [comment]
@tomdele please elaborate on the general policy of using this new util function `ensure_ids`. When it should be used? I still see some usages like `np.asarray(self._frame_population.get_node_ids(), dtype=np.int64)` in `frame_report.py` or `np.asarray(selection.flatten(), dtype=np.int64)` in `edges.py`.

**Comment by tomdele** [comment]
I totally forgot the reports indeed. 
I tried to explain a bit better the policy. Let me know if this is clearer.

File: bluepysnap/utils.py [hunk 1]
```diff
@@ -72,6 +73,8 @@ def ensure_ids(a):
     It is quite unsafe to the use uint64 for the ids due to this problem where :
     numpy.uint64 + int --> float64
     numpy.uint64 += int --> float64
+
+    This function needs to be used everywhere node_ids or edge_ids are returned.
```
Comment (asanin-epfl) [inline bluepysnap/utils.py:77]: Now this is very clear :D

**Review by asanin-epfl — APPROVED** [review]

**Review by asanin-epfl — COMMENTED** [review]
Forgot one thing. Should be CHANGELOG.rst updated?

**Comment by tomdele** [comment]
> Forgot one thing. Should be CHANGELOG.rst updated?

Hopefully you are here @asanin-epfl !

**Review by asanin-epfl — APPROVED** [review]

**Comment by tomdele** [comment]
Thank you !

**Pull request merged**